shared_model/interfaces/common_objects/types.hpp

```diff
@@ -48,6 +48,8 @@ namespace shared_model {
       using HeightType = uint64_t;
       /// Type of peer address
       using AddressType = std::string;
+      /// Type of peer address
+      using AddressTypeView = std::string_view;
```

Comment:
Why create an alias instead of strong type? Check how PublicKeyStringView is defined.

Reply:
I saw. Than ```AddressType``` will be alias for ```std::string``` and ```AddressTypeView```. It will redundant until ```AddressType``` not a strong type.